libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -199,6 +199,11 @@ enum PaymentTypeFilter {
     "ClosedChannel",
 };
 
+dictionary PaymentMetadataFilter {
```

Comment:
NIT: rename to `MetadataFilter` ? This looks generic enough to apply (if needed) later on other entities.